packages/nodes-base/nodes/Zammad/descriptions/TicketDescription.ts

```diff
@@ -0,0 +1,588 @@
+import {
+	INodeProperties,
+} from 'n8n-workflow';
+
+export const ticketDescription: INodeProperties[] = [
+	// ----------------------------------
+	//           operations
+	// ----------------------------------
+	{
+		displayName: 'Operation',
+		name: 'operation',
+		type: 'options',
+		noDataExpression: true,
+		displayOptions: {
+			show: {
+				resource: [
+					'ticket',
+				],
+			},
+		},
+		options: [
+			{
+				name: 'Create',
+				value: 'create',
+				description: 'Create a ticket',
+			},
+			{
+				name: 'Delete',
+				value: 'delete',
+				description: 'Delete a ticket',
+			},
+			{
+				name: 'Get',
+				value: 'get',
+				description: 'Retrieve a ticket',
+			},
+			{
+				name: 'Get All',
+				value: 'getAll',
+				description: 'Retrieve all tickets',
+			},
+			{
+				name: 'Update',
+				value: 'update',
+				description: 'Update a ticket',
+			},
+		],
+		default: 'create',
+	},
+
+	// ----------------------------------
+	//             fields
+	// ----------------------------------
+	{
+		displayName: 'Title',
+		name: 'title',
+		type: 'string',
+		description: 'Title of the ticket to create',
+		default: '',
+		required: true,
+		displayOptions: {
+			show: {
+				resource: [
+					'ticket',
+				],
+				operation: [
+					'create',
+				],
+			},
+		},
+	},
+	{
+		displayName: 'Group Name/ID',
+		name: 'group',
+		type: 'options',
+		typeOptions: {
+			loadOptionsMethod: 'loadGroupNames',
+		},
+		placeholder: 'First-Level Helpdesk',
+		description: 'Group that will own the ticket to create. Choose from the list or specify an ID using an <a href="https://docs.n8n.io/nodes/expressions.html#expressions">expression</a>.',
+		default: '',
+		required: true,
+		displayOptions: {
+			show: {
+				resource: [
+					'ticket',
+				],
+				operation: [
+					'create',
+				],
+			},
+		},
+	},
+	{
+		displayName: 'Customer Email',
```

Comment:
- Because the entities exist in a fetchable number. Unlike an overly large number like Elasticsearch docs.
- Because the user can easily pick the entity, for better UX.
- Because the guide does not forbid this.